libr/core/cmd_open.c

```diff
@@ -210,6 +210,7 @@ static void list_maps_visual(RIO *io, ut64 seek, ut64 len, int width, int use_co
 			for (j=0; j<width; j++) {
 				ut64 pos = min + (j*mul);
 				ut64 npos = min + ((j+1)*mul);
```

Comment:
missing spaces around each operator

Reply:
I believe there are other issues here. But I haven't touched them so I don't want to fix them in this PR and I do not have time.